cscs-checks/microbenchmarks/gpu/gpu_burn/gpu_burn_test.py

```diff
@@ -142,20 +143,19 @@ def report_smallest_node(self):
         rptf = os.path.join(self.stagedir, sn.evaluate(self.stdout))
         self.nids = sn.extractall(regex, rptf, 1)
         self.flops = sn.extractall(regex, rptf, 2, float)
-        index = -1
-        flops_min = sn.min(self.flops)
-        for ii in range(len(sn.evaluate(self.flops))):
-            if self.flops[ii] == flops_min:
-                index = ii
-                break
+        flops_min, index = sys.float_info.max, None
+        for i, flops in enumerate(self.flops):
+            if flops < flops_min:
+                flops_min, index = flops, i
 
         self.unit = f'GF/s ({self.nids[index]})'
+        unit = (0, None, None, self.unit)
         self.perf_patterns['smallest_flops'] = flops_min
-        self.reference['dom:gpu:smallest_flops'] = (0, None, None, self.unit)
-        self.reference['daint:gpu'] = (0, None, None, self.unit)
-        self.reference['arolla:cn'] = (0, None, None, self.unit)
-        self.reference['tsa:cn'] = (0, None, None, self.unit)
-        self.reference['ault:amda100'] = (0, None, None, self.unit)
-        self.reference['ault:amdv100'] = (0, None, None, self.unit)
-        self.reference['ault:intelv100'] = (0, None, None, self.unit)
-        self.reference['ault:amdvega'] = (0, None, None, self.unit)
+        self.reference['dom:gpu:smallest_flops'] = unit
```

Comment:
I think if you only put a single entry, it will print out the unit for all systems.

Reply:
All I meant is that this is enough. You don't need to set the reference here and no need to touch the references in the `__init__()`.